### ISSUE TITLE
PySB assembled model and BNG error

### ISSUE THREAD

**Issue by bgyori** [sorgerlab/indra#113]
The PySB model assembled for `KRAS activates BRAF` is
```
# exported from PySB model 'None'

from pysb import Model, Monomer, Parameter, Expression, Compartment, Rule, Observable, Initial, MatchOnce, Annotation, ANY, WILD

Model()

Monomer('RAF1', [u'act'], {u'act': [u'inactive', u'active']})
Monomer('KRAS', [u'act'], {u'act': [u'inactive', u'active']})

Parameter(u'kf_kr_act_1', 1e-06)
Parameter('RAF1_0', 100.0)
Parameter('KRAS_0', 100.0)

Rule(u'KRAS_activity_activates_RAF1_activity', KRAS(act=u'active') + RAF1(act=u'inactive') >> KRAS(act=u'active') + RAF1(act=u'active'), kf_kr_act_1)

Initial(RAF1(act=u'inactive'), RAF1_0)
Initial(KRAS(act=u'inactive'), KRAS_0)

Annotation(RAF1, u'http://identifiers.org/uniprot/Q06891', u'is')
Annotation(KRAS, u'http://identifiers.org/uniprot/P01116', u'is')
```
when doing
```
from pysb import bng
bng.generate_equations(model)
```
I get 
```BngInterfaceError: BNG generator has encountered an unknown element in a rule pattern site condition.```

**Comment by bgyori** [comment]
So I realize this is because of
https://github.com/pysb/pysb/blob/master/pysb/generator/bng.py#L183

**Comment by bgyori** [comment]
This is fixed on the INDRA end in 6818fe5513c2af3c57b316b9ac5aea7b2828465d and the corresponding PySB fix is still under way. Closing it here.
